core/src/main/java/net/adamcin/oakpal/core/checks/Paths.java

```diff
@@ -125,6 +134,14 @@ public ProgressCheck newInstance(final JSONObject config) throws Exception {
         final boolean denyAllDeletes = config.has(CONFIG_DENY_ALL_DELETES)
                 && config.optBoolean(CONFIG_DENY_ALL_DELETES);
 
-        return new Check(rules, denyAllDeletes);
+        Violation.Severity severity = null;
+        try {
+            if (config.has(CONFIG_SEVERITY)) {
+                severity = Violation.Severity.valueOf(config.optString(CONFIG_SEVERITY).toUpperCase());
+            }
+        } catch (Exception e) {
```

Comment:
You might as well lose the try-catch here. The ProgressCheckFactory.newInstance() method already throws Exception for arbitrary errors.